README.md

```diff
@@ -5,45 +5,164 @@
+  
+### **fuse.eval** - a standalone library for **evaluating ML models** (not necessarily trained with FuseMedML)  
+The package includes collection of off-the-shelf metrics and utilities such as **statistical significance tests, calibration, thresholding, model comparison** and more.  
+See detailed introduction [here](https://github.com/IBM/fuse-med-ml/tree/master/fuse/eval/README.md)  
+
+## Supported DL libraries
+Some components depend on pytorch. For example, ```fuse.data``` is oriented towards pytorch DataSet, DataLoader, DataSampler etc.
+```fuse.dl``` makes heavy usage of pytorch models.
+Some components do not depend on any specific DL library - for example ```fuse.eval```.
+
+Broadly speaking, the supported DL libraries are:
+* "Pure" [pytorch](https://pytorch.org/)
+* [pytorch-lightning](https://www.pytorchlightning.ai/)
+
+Before you ask - **pytorch-lightning and FuseMedML play along very nicely and have in practice orthogonal and additive benefits :)**  
+See [Simple FuseMedML + PytorchLightning Example](./examples/fuse_examples/imaging/classification/mnist/run_mnist.py) for simple supervised learning cases, and [this example ](./examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py) for completely custom usage of pytorch-lightning and FuseMedML - useful for advanced scenarios such as Reinforcement Learning and generative models.
+
+## Domain Extensions
+fuse-med-ml, the core library, is completely domain agnostic!
+Domain extensions are optionally installable packages that deal with specific (sub) domains. For example:
+
+* **fuseimg** which was battletested in many medical imaging related projects (different organs, imaging modalities, tasks, etc.)
+* **fusedrug** which focuses on molecular biology and chemistry - prediction, generation and more
+
+Domain extensions contain concrete implementation of components and components parts within the relevant domain, for example:  
+* Data pipeline operations - for example, a 3d affine transformation of a 3d image
+* Evaluation metrics - for example, a custom metric evaluating docking of a potential drug with a protein target
+* Loss functions - for example, a custom segmentation evaluation loss
+
+The recommended directory structure mimics fuse-med-ml core structure
+```
+your_package
+    data #everything related to datasets, samplers, data processing pipeline Ops, etc.
+    dl #everything related to deep learning architectures, optimizers, loss functions etc.
+    eval #evaluation metrics
+    utils #any utilities
+```
+
+You are highly encouraged to create additional domain extensions and/or contribute to the existing ones!
+There's no need to wait for any approval, you can create domain extensions on your own repos right away
+
+Note - in general, we find it helpful to follow the same directory structure shown above even in small and specific research projects that use FuseMedML for consistency and easy landing for newcomers into your project :)
```

Comment:
Do we have examples of such projects that depend on fuse but are managed independently?